torch/onnx/utils.py

```diff
@@ -76,7 +77,7 @@ def export(model, args, f, export_params=True, verbose=False, training=None,
     if aten or export_raw_ir:
         assert operator_export_type is None
         assert aten ^ export_raw_ir
-        operator_export_type = OperatorExportTypes.ATEN if aten else OperatorExportTypes.RAW
+        operator_export_type = OperatorExportTypes.ONNX_ATEN if aten else OperatorExportTypes.RAW
```

Comment:
There is no mention to `OperatorExportTypes.ATEN` in the source code. So, I think this value was supposed to be `OperatorExportTypes.ONNX_ATEN`.